src/practitest_firecracker/core.clj

```diff
@@ -24,37 +24,45 @@
        (println (str ~module " elapsed time: " (/ (double (- (. System (nanoTime)) start#)) 1000000.0) " msecs")))
      ret#))
 
+(defn clean-tmp-folder [directory]
+  (doseq [path directory]
+    (delete-recursively! path)))
```

Comment:
Did you test this part? Are you sure `directory` is an array of paths?

Reply:
it works, I tested it.
directory comes from: 
`directory          (map #(.getAbsolutePath (file (:temp-folder options) %)) (:reports-path options))`